### PR TITLE
*: release v2.0.0-rc.0

### PR DESCRIPTION
**Pull request by Gouthamve**
@fabxc @grobie @brian-brazil

---
This change is [Reviewable](https://reviewable.io/reviews/prometheus/prometheus/3248)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+
+This release includes numerous changes to the new storage layer. The main changes are:
+
+* [CHANGE] Remove count_scalar, keep_common and drop_common_labels functions
+* [BUGFIX] Fix panic due garbage collected mmap'ed strings 
+* [BUGFIX] Fix broken snapshots and admin APIs 
+* [BUGFIX] Send HTTP Accept header when scraping
+* [BUGFIX] Use the WAL flush interval passed instead of the hardcoded value
+
+This release requires a clean storage directory and is not compatible with files
```
Comment (grobie) [inline CHANGELOG.md]: I'd expect another `[CHANGE] ...` line for the disk format changes.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-2.0.0-beta.5
+2.0.0-rc.1
```
Comment (fabxc) [inline VERSION]: `rc.0`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## v2.0.0-rc.1 / 2017-10-05
+
+This release includes numerous changes to the new storage layer. The main changes are:
+
+* [CHANGE] Remove count_scalar, keep_common and drop_common_labels functions
```
Comment (fabxc) [inline CHANGELOG.md]: Function names in backticks for prettyness.

**Comment by fabxc** [comment]
👍 otherwise

**Comment by Gouthamve** [comment]
@grobie @fabxc Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
+## v2.0.0-rc.0 / 2017-10-05
+
+This release includes numerous changes to the new storage layer. The main changes are:
+
+* [CHANGE] Remove `count_scalar`, `keep_common` and `drop_common_labels` functions
```
Comment (aecolley) [inline CHANGELOG.md]: Also `delta`.

**Comment by aecolley** [comment]
This has been running fine for me with production traffic. I'd be happy if this became the 2.0.0 release.

**Pull request merged**